04_numeronym/selva/main_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var nemeronymsTest = []struct {
+	in  []string
+	out []string
+}{
+	{[]string{"accessibility", "internationalisation"}, []string{"a11y", "i18n"}},
```

Comment:
Nice!
Add empty slice, empty string, string with unicodes in middle and at end, strings with white space in middle and end.

Reply:
Added more cases.